packages/components/radio/src/Radio.variants.ts

```diff
@@ -0,0 +1,46 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+const defaultVariants = {
+  intent: 'primary',
+  size: 'small',
+} as const
+
+export const radioVariants = cva(
+  [
+    'block',
+    'rounded-full',
+    'border-md',
+    'border-outline',
+    'outline-transparent',
+    'rounded-full',
+    'hover:outline',
+    'hover:outline-4',
+    'focus-visible:outline',
+    'focus-visible:outline-1',
+    'focus-visible:outline-on-surface',
+    '[transition:outline-color_0.1s_ease-in,_border-color_0.3s_ease-in]',
+    'disabled:border-outline/50',
+    'disabled:hover:outline-none',
+    'disabled:cursor-not-allowed',
+  ],
+  {
+    variants: {
+      size: {
+        small: ['h-[1.25rem]', 'w-[1.25rem]'],
```

Comment:
we should avoid using arbitrary values, as it may lead to UI inconsistencies if a different base font size is used at the call site

Reply:
Yes! sorry I forgot to make the change

Reply:
Changes made! thanks @acd02